test/parallel/test-options-binding.js

```diff
@@ -0,0 +1,21 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+const { primordials } = require('internal/test/binding');
+const {
+  SafeMap,
+} = primordials;
+
+const { options, aliases, getOptionValue } = require('internal/options');
+const assert = require('assert');
+
+assert(options instanceof SafeMap,
+       "require('internal/options').options is a SafeMap");
+
+assert(aliases instanceof SafeMap,
+       "require('internal/options').aliases is a SafeMap");
+
+Map.prototype.get =
+  common.mustNotCall('%Map.prototype.get% must not be called');
```

Comment:
+1 on using `mustNotCall` here, but the message is confusing to me. Souldn't `%Map.prototype.get%` be called, instead of `globalThis.Map.prototype.get`? Based on how the ECMAScript spec uses this notation at least (https://tc39.es/ecma262/#sec-well-known-intrinsic-objects).

```suggestion
  common.mustNotCall('`getOptionValue` must not be call user-mutable function');
```